master/getting-started/rkt/installation/vagrant-coreos/Vagrantfile

```diff
@@ -5,14 +5,17 @@ num_instances=2
 instance_name_prefix="calico"
 
 # Official CoreOS channel from which updates should be downloaded
-update_channel='stable'
+# Using the alpha channel because the rkt version shipped with stable doesn't have a rkt stop command,
+# and does not honor the config path during container detach (which means we'd leak IP addresses when
+# it can't locate the IPAM module.
+update_channel='alpha'
 
 Vagrant.configure("2") do |config|
   # always use Vagrants insecure key
   config.ssh.insert_key = false
 
   config.vm.box = "coreos-%s" % update_channel
-  config.vm.box_version = ">= 1122.0.0"
+  config.vm.box_version = ">= 1235.1.0"
```

Comment:
you updated the `update_channel` but the version is still set to the one for `beta`. This should be `1248.1.0`